harmonica/_io/geosoft_gxf_io.py

```diff
@@ -0,0 +1,352 @@
+    
+    # Handle grid orientation based on SENSE parameter
+    sense = str(headers.get('SENSE', '1'))  # Default to '1' if not specified
+    if sense == '1':  # First point at bottom left of grid
+        grid_array = np.flipud(grid_array)
+    
+    # Add convenient grid parameters
+    metadata.update({
+        'nx': ncols,
+        'ny': nrows,
+        'x_inc': float(headers.get('PTSEPARATION', headers.get('XSEP', 1.0))),
+        'y_inc': float(headers.get('RWSEPARATION', headers.get('YSEP', 1.0))),
+        'x_min': float(headers.get('XORIGIN', 0.0)),
+        'y_min': float(headers.get('YORIGIN', 0.0))
+    })
+    
+    # Add projection information if available
+    if 'PRJTYPE' in headers:
+        metadata['projection'] = {
+            'type': headers['PRJTYPE'],
+            'units': headers.get('PRJUNIT', 'unknown'),
+            'parameters': {
+                'semi_major_axis': float(headers['A_AXIS_RADIUS']) if 'A_AXIS_RADIUS' in headers else None,
+                'semi_minor_axis': float(headers['B_AXIS_RADIUS']) if 'B_AXIS_RADIUS' in headers else None,
+                'reference_longitude': float(headers['RFLONGITUDE']) if 'RFLONGITUDE' in headers else None,
+                'reference_latitude': float(headers['RFLATITUDE']) if 'RFLATITUDE' in headers else None,
+                'first_standard_parallel': float(headers['FIRST_STANDARD_PARALLEL']) if 'FIRST_STANDARD_PARALLEL' in headers else None,
+                'second_standard_parallel': float(headers['SECOND_STANDARD_PARALLEL']) if 'SECOND_STANDARD_PARALLEL' in headers else None,
+                'false_easting': float(headers['FLSEASTING']) if 'FLSEASTING' in headers else None,
+                'false_northing': float(headers['FLSNORTHING']) if 'FLSNORTHING' in headers else None
+            }
+        }
+    
+    return grid_array, metadata
+
+def get_grid_info(metadata: Dict[str, Any]) -> None:
+    """
+    Print comprehensive information about the GXF grid.
+    
+    Parameters:
+    metadata (dict): Metadata dictionary from read_gxf
+    """
+    print("=== Grid Information ===")
+    print(f"Title: {metadata.get('TITLE', 'Not specified')}")
+    print(f"Grid size: {metadata['nx']} x {metadata['ny']} points")
+    print(f"Cell size: {metadata['x_inc']} x {metadata['y_inc']}")
+    print(f"Origin: ({metadata['x_min']}, {metadata['y_min']})")
+    print(f"Rotation: {metadata.get('ROTATION', 0)}")
+    print(f"Dummy value: {metadata.get('DUMMY', 'Not specified')}")
+    
+    if 'projection' in metadata:
+        print("\n=== Projection Information ===")
+        print(f"Type: {metadata['projection']['type']}")
+        print(f"Units: {metadata['projection']['units']}")
+        
+        params = metadata['projection']['parameters']
+        if any(params.values()):
+            print("\nProjection Parameters:")
+            for key, value in params.items():
+                if value is not None:
+                    print(f"{key}: {value}")
```

Comment:
I think this function is nice for debugging the reader. But I hardly see users using it (we would need to expose it as well). Instead, if `read_gxf` returns a DataArray, then we don't need this function: `xarray` can nicely display all the data.

What do you think? Is there any special reason to include this function?